src/index.js

```diff
@@ -134,7 +134,7 @@ export function createEventLogger(options: EventLoggerOptionsType): Function {
       }
     }
   }
-  if (endpoint === null) {
+  if (endpoint === null || endpoint === undefined) {
```

Comment:
we actually had a test case fail where a 'boolean' was passed for 'endpoint' and that is why we moved away from !variable or !!variable in some cases